Kevin-1101-guessmyword.py

```diff
@@ -0,0 +1,117 @@
+    # -----------------------------------
+    # Helper code
+    # You don't need to understand this helper code,
+    # but you will have to know how to use the functions
+    # (so be sure to read the docstrings!)
+    
+    import random
+    import string
+    
+    WORDLIST_FILENAME = "word_list.txt"
+    '''
+    returns valid words
+        '''
+    def secret_words():
+    
+        print("Reading word_list file...")
+        # inFile: file
+        inFile = open(WORDLIST_FILENAME, 'r')
+        # line: string
+        line = inFile.readline()
+        # word_list: list of strings
+        word_list = line.split()
+        print(len(word_list), "words found")
+        return word_list
+    
+    def choose_word(word_list):
+        
+        
+        return random.choice(word_list)
+    
+    # end of helper code
+    # -----------------------------------
+    # Load the list of words into the variable word_list
+    # so that it can be accessed from anywhere in the program
+    word_list = secret_words()
+    '''
+    word is the word that the player has to guess
+    guess is the list of letters that the player has guessed so far 
+    
+    '''  
+    def is_word_guessed(word, guessed):
+       
+        # FILL IN YOUR CODE HERE...
+        for c in word:
+            if c not in guessed:
+                return False
+        return True
+    
+    
+    
+    
+    def get_guessed_word(word, guessed):
+        
+        
+        # FILL IN YOUR CODE HERE...
+        output = ""
+        for c in word:
+            if c in guessed: 
+                output += c
+            else:
+                output += " "

```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).